deepvariant/openvino_estimator.py

```diff
@@ -49,10 +49,11 @@ def is_available():
     return openvino_available
 
 
-  def __init__(self, checkpoint_path, num_channels, *, input_fn, model):
-    freeze_graph(model, checkpoint_path, num_channels)
+  def __init__(self, checkpoint_path, *, input_fn, model):
```

Comment:
Thank you @dkurt . FYI this commit has been reviewed internally and added now as well. It'll show up in our next release.

One question for you about this line: what is the "*" arg passed in here? I removed it and it worked fine. But wanted to check unless there's a reason it has to be there.

Reply:
@pichuan, thank you so much!

This is an arguments delimiter which makes `input_fn` and `model` are required to be named parameters. So
```
OpenVINOEstimator(checkpoint_path, input_fn=input_fn, model=model)
```